### ISSUE TITLE
fCNV dies on .cnv files with lines containing only "*"

### ISSUE THREAD

**Issue by dnowacki-usgs** [castelao/seabird#37]
Using seabird 0.8.0 on Anaconda 4.1.11 on OS X 10.10.5. I have data from a 19plusV2 processed using SBEDataProcessing 7.26.1.8. This version produces .cnv files with lines containing only "*" inside the `<InstrumentState>` tag. Example below:

```
* <HardwareData DeviceType='SBE19plus' SerialNumber='01906651'>
*
*    <Manufacturer>Sea-Bird Electronics, Inc.</Manufacturer>
*
*    <FirmwareVersion>2.2c</FirmwareVersion>
*
```

When trying to process a file like this, fCNV dies with the error `There are no rules able to parse the input. File: myfile.cnv`. 

Deleting all the lines containing only "*" in the cnv enables fCNV to parse the file without issue. I don't know enough about yaml rules to submit a fix, unfortunately.

**Comment by porterdf** [comment]
I'd like to chime in that I'm also having this problem.  

In addition, fCNV fails when there are blank lines.  Again, deleting these blank lines, as @dnowacki-usgs did for the lines containing only "*", worked.

**Comment by castelao** [comment]
Hi @dnowacki-usgs  and @porterdf,  thanks for the notes.

The branch dev was patched to handle your files. That should work fine for now, but I'll properly address that and upgrade the package in the first chance. Hopefully in the following days.

Thanks!

**Comment by porterdf** [comment]
Thanks @castelao!  BTW thanks for developing this super useful package.

**Comment by castelao** [comment]
Resolved in the latest official release, 0.10
